services/comfy/entrypoint.sh

```diff
@@ -44,4 +44,16 @@ for to_path in "${!MOUNTS[@]}"; do
   echo Mounted $(basename "${from_path}")
 done
 
+if [ -f "/data/config/comfy/startup.sh" ]; then
+  pushd ${ROOT}
+  . /data/config/comfy/startup.sh
+  popd
+fi
+
+shopt -s nullglob
+list=(${ROOT}/custom_nodes/*/requirements.txt)
+for req in "${list[@]}"; do
+  pip install -r "$req"
+done
+
```

Comment:
I do still think this should be separate, otherwise it could introduce so many bugs we cannot account for.
i.e. we build the container with certain dependencies, yet at startup they get overwritten by random ones from the extensions.

An example which is very frustrating that comes to mind is `open-cv-headless`.

Reply:
/output/comfy does exist on my system, yes.

How would you want to separate out that part?